certified-connectors/MongoDB/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1421 @@
+            "description": "Content-Type"
+          },
+          {
+            "name": "Access-Control-Request-Headers",
+            "x-ms-summary": "Access-Control-Request-Headers",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "*",
+            "description": "Access-Control-Request-Headers"
+          },
+          {
+            "name": "api-key",
+            "x-ms-summary": "api-key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "{{API_KEY}}",
+            "description": "api-key"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "dataSource": {
+                  "type": "string",
+                  "description": "datasource",
+                  "title": ""
+                },
+                "database": {
+                  "type": "string",
+                  "description": "database"
+                },
+                "collection": {
+                  "type": "string",
+                  "description": "collection"
+                },
+                "document": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "age": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "age"
+                    }
+                  },
+                  "description": "document"
+                }
+              },
+              "default": {
+                "dataSource": "{{DATA_SOURCE}}",
+                "database": "{{DATABASE}}",
+                "collection": "{{COLLECTION}}",
+                "document": {
+                  "name": "John Sample",
+                  "age": 42
+                }
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "A JSON object containing objectid of the inserted document.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "insertedId": {
+                  "type": "string",
+                  "description": "insertedId"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {},
```

Comment:
@DianaMDB empty schemas are [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).